project_category/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 ADHOC SA (http://www.adhoc.com.ar)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Is it possible to make all the license links secure http://www.gnu.org/licenses/agpl. We have already converted them in other OCA repositories. Thank you

Reply:
in other OCA repositories?
Please, tell me an example

Reply:
https://github.com/OCA/crm/pull/331

Reply:
ok, done